src/content/docs/change-tracking/change-tracking-events.mdx

```diff
@@ -150,7 +150,7 @@ To record deployments and other types of change events, use the NerdGraph mutati
 Use the <DNT>`changeTrackingCreateEvent`</DNT> mutation to create a change event by providing its required and optional attributes.
 
 <Callout variant="important">
```

Comment:
with [NR-417056](https://new-relic.atlassian.net/browse/NR-417056) the APM Summary Deployments will be showing accurate results.